massa-proto/proto/massa/api/v1/operation.proto

```diff
@@ -63,10 +63,12 @@ message RollSell {
 message ExecuteSC {
   // Smart contract bytecode.
   bytes data = 1;
+  // The maximum of coins that could be spent by the operation sender
```

Comment:
In the mainet, we couldn't do this, we always have to add the end of type to avoid breaking changes.

Reply:
Ok but for this month it's still ok to do this ?